src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx

```diff
@@ -50,8 +55,8 @@ export default async function AutomaticRemovePage() {
       subscriberEmails={subscriberEmails}
       nextStep={getNextGuidedStep(data, "Scan")}
       currentSection="data-broker-profiles"
-      monthlySubscriptionUrl={monthlySubscriptionUrl}
-      yearlySubscriptionUrl={yearlySubscriptionUrl}
+      monthlySubscriptionUrl={`${monthlySubscriptionUrl}&${additionalSubplatParams.toString()}`}
```

Comment:
It might be easier/safer to use `searchParams` instead of appending strings like this... I think it's OK but something to consider.

Reply:
Not as straightforward, since the existing subscriptionUrls already have existing `?` and `&` (search params). So it would mean we'd have to deconstruct that url etc